docs/testing.md

```diff
@@ -103,6 +112,12 @@ sudo docker run hello-world
   :warning: NOTE: For more details about Python virtual environment see
   https://docs.python.org/3.6/library/venv.html
 
+- Activate created Python environment
+
+  ```bash
+  source venv/bin/activate
+  ```
+
```

Comment:
Could you  please also update the line 162: currently the maximum number of [nodes is 6](https://github.com/Seagate/cortx-prvsnr/blob/2ebd2bf82bc349a724ae39346f464f20a4c70009/test/conftest.py#L45)

Reply:
Yes, sure. I updated the parameter description.